platform/security/sts/security-sts-casclaimshandler/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -4,25 +4,29 @@
  * Copyright (c) Codice Foundation
  *
  * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
- * version 3 of the License, or any later version. 
+ * version 3 of the License, or any later version.
  *
  * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
  * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  *
  **/
+-->
+<blueprint xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+  xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
 
- -->
-<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+  <bean id="casHandler" class="org.codice.ddf.security.claims.cas.CasClaimsHandler">
+    <cm:managed-properties
+      persistent-id="ddf.security.claims.cas.CasClaimsHandler"
+      update-strategy="container-managed"/>
+    <property name="attributeMap">
+      <map/>
+    </property>
+    <property name="supportedClaims">
```

Comment:
❓ does this property need to exist if we can get the claims from the attributemap keys?